cas-server-webapp/pom.xml

```diff
@@ -79,6 +79,11 @@
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-web</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.pac4j</groupId>
+      <artifactId>pac4j-http</artifactId>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
Perfect. I think you may want to exclude them all, because CAS explicitly declares all 3 of those. (not sure about codec)